gatewayd.yaml

```diff
@@ -32,33 +32,57 @@ metrics:
 
 clients:
   default:
-    network: tcp
-    address: localhost:5432
-    tcpKeepAlive: False
-    tcpKeepAlivePeriod: 30s # duration
-    receiveChunkSize: 8192
-    receiveDeadline: 0s # duration, 0ms/0s means no deadline
-    receiveTimeout: 0s # duration, 0ms/0s means no timeout
-    sendDeadline: 0s # duration, 0ms/0s means no deadline
-    dialTimeout: 60s # duration
-    # Retry configuration
-    retries: 3 # 0 means no retry and fail immediately on the first attempt
-    backoff: 1s # duration
-    backoffMultiplier: 2.0 # 0 means no backoff
-    disableBackoffCaps: false
+    activeWrites:
```

Comment:
The fact this node appears let me think (I might be wrong) this is a breaking changes.

Is it true? Ia everyone OK with that?

Reply:
This changes is backward incompatible by design. We can opt for introducing backward compatibility, but it will increase maintenance overhead in the long run. I'll add a warning to the release notes for this.